### PR TITLE
Fix/select box

### PR DESCRIPTION
**Pull request by Crash--**
- Menu => ActionMenu on Mobile 
- Use Avatar from cozy-ui (need PR from UI https://github.com/cozy/cozy-ui/pull/648) 
- Use popover props from UI ( https://github.com/cozy/cozy-ui/pull/647) 
- Styles correction. Not happy with my `override class` ..

### CODE REVIEW

File: src/sharing/components/Recipient.jsx [hunk 1]
```diff
@@ -113,23 +116,33 @@ class Status extends Component {
...
                 ? t(`Share.type.${type}`)
                 : t(`Share.status.${status}`)
             }
             className={styles['recipient-menu']}
             buttonClassName={styles['recipient-menu-btn']}
-            position={isMobile ? 'left' : 'right'}
+            position={'right'}
+            popover
+            itemsStyle={{
+              width: '272px'
```
Comment (y-lohse) [inline src/sharing/components/Recipient.jsx]: Do we really want this as a fixed size?

File: src/sharing/share.styl [hunk 2]
```diff
@@ -54,6 +54,9 @@
     margin 0
     border-top 1px solid silver
 
+.share-moral-override-bottom
+    padding-bottom : 0 !important
```
Comment (y-lohse) [inline src/sharing/share.styl]: I that here to override a padding from cozy-ui?

File: src/components/Menu/Menuawaremobile.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      return (
+        <div>
+          <MenuButton
+            onClick={this.toggle}
+            text={text}
+            buttonClassName={buttonClassName}
+          />
+          {this.state.active && (
+            <ActionMenu onClose={() => this.toggle()}>
+              <MenuItem className={styles['fil-mobileactionmenu']}>
```
Comment (y-lohse) [inline src/components/Menu/Menuawaremobile.jsx]: Does the action menu have a title in the end?

Comment (Crash--) [reply src/components/Menu/Menuawaremobile.jsx]: Nop. Only items.

**Comment by Crash--** [comment]
[(3) PARTAGE - Problème affichage dropdown](https://trello.com/c/G8O8m1fY/1194-3-partage-probl%C3%A8me-affichage-dropdown)

File: src/drive/web/modules/drive/Toolbar/share/ShareItem.jsx [hunk 4]
```diff
@@ -19,7 +19,7 @@ const ShareItem = translate()(
               className={styles['fil-toolbar-menu-recipients']}
               recipients={recipients}
               link={link}
-              size="small"
+              size="small-plus"
```
Comment (y-lohse) [inline src/drive/web/modules/drive/Toolbar/share/ShareItem.jsx]: This size doesn't match anything at the moment, is it really what we want?

**Comment by Crash--** [comment]
@y-lohse squashed & rebased.

**Pull request merged**